docs/user_guide/authenticate.md

```diff
@@ -1,11 +1,157 @@
 # Authentication
 
-Introduces the `earthaccess.login` method for managing Earthdata Login and cloud credentials.
+You can use `earthaccess` to search for datasets and data without needing to log in.
+However, to access (download or stream) NASA Earth science data, whether from one of the NASA
+Distributed Active Archive Centers (DAACs) or from NASA Earthdata Cloud, you need
```

Comment:
```suggestion
Distributed Active Archive Centers (DAACs) on-premises archive or from NASA Earthdata Cloud, you need
```

Reply:
data from NASA Earthdata Cloud still come from the DAAC. Just clarifying so that it doesn't sound like an either-or.